src/Symfony/Component/FeatureFlag/Exception/FeatureNotFoundException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlag\Exception;
+
+class FeatureNotFoundException extends \RuntimeException
```

Comment:
what about creating a component base exception ?
it helps after to make subscriber/listener to catch such

Reply:
Good idea. It's done.